odoo/tools/translate.py

```diff
@@ -1475,6 +1475,10 @@ def __init__(self):
     @staticmethod
     def _get_po_paths(mod, lang):
         lang_base = lang.split('_')[0]
+        if lang_base == "es" and lang != "es_MX":
+            # force es_MX as fallback language for the different spanish
+            # es_MX is more actively translated and closer to many languages
+            lang_base = "es_MX"
         po_paths = [get_resource_path(mod, 'i18n', lang_base + '.po'),
```

Comment:
@HydrionBurst why is `lang_base` before `lang` here? Shouldn't it be the opposit?

Reply:
we import code translations for `lang_base` first, and then import translations for `lang` to override the translations for `lang_base`